Makefile

```diff
@@ -505,7 +505,7 @@ publish-artifacts: opscenter telekube
 # scan-artifacts uploads a copy of all vendored containers to a docker registry for scanning and vulnerability reporting
 #
 .PHONY: scan-artifacts
-scan-artifacts:
+scan-artifacts: telekube
```

Comment:
Including a fix for https://jenkins.gravitational.io/view/Publishing/job/Gravity-Publish-5.2/ failing to publish the artifacts to s3.